statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1109,28 +1161,39 @@ def __init__(self, endog, endog_lagged, params, sigma_u, lag_order,
         # Each matrix needs to be transposed
         endog_start = k_trend
         if exog is not None:
-            endog_start += exog.shape[1]
+            k_exog_user = exog.shape[1]
+            endog_start += k_exog_user
+        else:
+            k_exog_user = 0
         reshaped = self.params[endog_start:]
         reshaped = reshaped.reshape((lag_order, neqs, neqs))
         # Need to transpose each coefficient matrix
         coefs = reshaped.swapaxes(1, 2).copy()
 
         self.coefs_exog = params[:endog_start].T
-        self.k_trend = self.coefs_exog.shape[1]
-
-        # print(coefs.round(3))
-        super(VARResults, self).__init__(coefs, exog, sigma_u, names=names)
+        #self.k_trend = self.coefs_exog.shape[1]
+        self.k_exog = self.coefs_exog.shape[1]
+        self.k_exog_user = k_exog_user
+
+        # maybe change to params class, distinguish exog_all versus exog_user
```

Comment:
Can you add "TODO: " to the beginning of this comment?

Reply:
BTW `VARParams` class [here](https://github.com/jbrockmendel/sm2/blob/master/sm2/tsa/wold.py#L705) isn't quite what I think you're referring to (which I think is more analogous to MixedLMParams), but may still be useful.